packages/webpack/package.json

```diff
@@ -70,6 +70,9 @@
     "@nx/devkit": "file:../devkit",
     "@nx/js": "file:../js"
   },
+  "peerDependencies": {
+    "browserslist": "^4.21.4"
```

Comment:
I would move this to `dependencies` (we used to have it  but removed it when we removed support for legacy browsers), or just load it with a try-catch, so it's completely optional.